blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -79,6 +80,9 @@ public void startServer() {
                         Executors.newFixedThreadPool(workerThreads)));
         server.setPipelineFactory(new MetricsHttpServerPipelineFactory(router));
         serverChannel =  (ServerChannel) server.bind(new InetSocketAddress(httpQueryHost, httpQueryPort));
+
+        log.info("Starting tracker service");
+        Tracker.register();
```

Comment:
When you call `register()` from both these place, whats happen differently between the calls. Inside the method, both these calls seem to be doing the same thing. If there is no difference, can you call `register()` only in the constructor of Tracker?